usmqe_tests/ui/test_volume_ui.py

```diff
@@ -0,0 +1,385 @@
+        """
+        :step:
+          For each volume calculate the expected subvolume/replica set size
+          from the volume name depending on volume type.
+        :result:
+          Subvolume/replica set size is calculated.
+        """
+        if volume.volname.split("_")[2] in {"arbiter", "disperse"}:
+            part_size = int(volume.volname.split("_")[3]) + \
+                        int(volume.volname.split("_")[5].split('x')[0])
+        elif volume.volname.split("_")[2] == "distrep":
+            part_size = int(volume.volname.split("_")[3].split('x')[1])
+        else:
+            pytest.check(False,
+                         "Volume type isn't ``arbiter``, ``distrep`` or ``disperse``")
+            LOGGER.debug("Unexpected volume type")
+            part_size = 0
+        """
+        :step:
+          For each volume part get its bricks.
+          Check that actual brick count per subvolume/replica set is as expected.
+        :result:
+          Each subvolume/replica set has the expected number of bricks.
+        """
+        volume_parts = volume.parts.get_parts()
+        pytest.check(volume_parts != [],
+                     "Check that there is at least one subvolume/replica set")
+        all_bricks = []
+        for part in volume_parts:
+            bricks = part.bricks.get_bricks()
+            pytest.check(len(bricks) == part_size,
+                         "Check that the number of bricks in the table is as expected")
+            for brick in bricks:
+                """
+                :step:
+                  Check each brick's attributes.
+                :result:
+                  All brick's attributes are as expected.
+                """
+                pytest.check(brick.brick_path.find('/mnt/brick') == 0,
+                             "Check that Brick Path starts with ``/mnt/brick``")
+                pytest.check(brick.utilization.find('% U') > 0,
+                             "Check that Utilization column values inclusde ``%``")
+                pytest.check(brick.disk_device_path.find('/dev/') == 0,
+                             "Check that Disk Device Path starts with ``/dev/``")
+                pytest.check(int(brick.port) > 1000,
+                             "Check that Port number is an integer greater than 1000")
+            all_bricks = all_bricks + bricks
+        """
+        :step:
+          Check that the list of all bricks of the volume in UI is the same as the result
+          of gluster volume info command.
+        :result:
+          The list of all volume's bricks in the UI is correct.
+        """
+        glv_cmd = gluster.GlusterVolume(volume_name=volume.volname)
+        glv_cmd.info()
+        LOGGER.debug("Gluster bricks: {}".format(glv_cmd.bricks))
+        ui_brick_names = [b.hostname + ":" + b.brick_path for b in all_bricks]
+        LOGGER.debug("UI bricks: {}".format(ui_brick_names))
+        pytest.check(glv_cmd.bricks == ui_brick_names,
```

Comment:
```suggestion
        pytest.check(sorted(glv_cmd.bricks) == sorted(ui_brick_names),
```
It seems to me that there is no guarantee that the ordering will be the same.